sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/VerifyResult.java

```diff
@@ -0,0 +1,23 @@
+package com.azure.security.keyvault.keys.cryptography;
+
+/**
+ * Represents the details of verify operation result.
+ */
+public class VerifyResult {
+    public VerifyResult(Boolean isValid) {
+        this.isValid = isValid;
+    }
+
+    /**
+     * THe verify operation result.
+     */
+    private Boolean isValid;
```

Comment:
Same statement as Srikanta, shouldn't this be the first declaration in the class

Reply:
updated